src/librustc_mir/build/mod.rs

```diff
@@ -341,6 +341,11 @@ impl<'a,'tcx> Builder<'a,'tcx> {
             }
         }
     }
+
+    pub fn check_overflow(&self) -> bool {
```

Comment:
What's confusing about it? I mean, it could be a little more verbose, but it's just so you don't have to type out that long expression. Given the context of its use, I'm not sure how else you'd interpret the meaning.